### PR TITLE
Attempt for packaging 3.0.0

### PR DESCRIPTION
**Pull request by FrancescAlted**
@conda-forge-admin , please rerender

Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,41 +23,49 @@ requirements:
     - python                              # [build_platform != target_platform]
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
     - cython                              # [build_platform != target_platform]
-    - numpy                               # [build_platform != target_platform]
+    - numpy >=2.0.0                       # [build_platform != target_platform]
```
Comment (hmaarrfk) [inline recipe/meta.yaml:26]: Please leave the numpy version in the build and host section blank. 

Conda forge manages build time pinnings in a global way

**Comment by hmaarrfk** [comment]
Converting to draft to avoid accidental merging. 

We don’t really publish rcs. (Well you can but the process is more complicated than one would want) 

good to know that this works!!!!

File: recipe/meta.yaml (hunk 1 above)
Comment (FrancescAlted) [reply recipe/meta.yaml]: Fixed in 1018cf3.  Thanks!

**Comment by hmaarrfk** [comment]
> could deploy conda-forge packages relatively smoothly

awesome thanks for your support!

**Comment by FrancescAlted** [comment]
FWIW, after not requiring numpy 2.0, the Python 3.13 builds are complaining:

```
The following packages are incompatible
├─ numpy =1.26 * is installable with the potential options
│  ├─ numpy [1.26.0|1.26.2|1.26.3|1.26.4] would require
│  │  └─ python_abi ==3.9 *_pypy39_pp73, which requires
│  │     └─ python =3.9 *_73_pypy, which can be installed;
│  ├─ numpy [1.26.0|1.26.2|1.26.3|1.26.4] would require
│  │  └─ python >=3.10,<3.11.0a0 *, which can be installed;
│  ├─ numpy [1.26.0|1.26.2|1.26.3|1.26.4] would require
│  │  └─ python >=3.11,<3.12.0a0 *, which can be installed;
│  ├─ numpy [1.26.0|1.26.2|1.26.3|1.26.4] would require
│  │  └─ python_abi =3.12 *_cp312, which requires
│  │     └─ python =3.12 *_cpython, which can be installed;
│  └─ numpy [1.26.0|1.26.2|1.26.3|1.26.4] would require
│     └─ python >=3.9,<3.10.0a0 *, which can be installed;
└─ python =3.13 *_cp313 is not installable because there are no viable options
   ├─ python [3.13.0|3.13.1] conflicts with any installable versions previously reported;
   └─ python [3.13.0rc1|3.13.0rc2|3.13.0rc3] would require
      └─ _python_rc =* *, which does not exist (perhaps a missing channel).
```

**Comment by FrancescAlted** [comment]
> FWIW, after not requiring numpy 2.0, the Python 3.13 builds are complaining:
> 
> ```
> The following packages are incompatible
> ├─ numpy =1.26 * is installable with the potential options
> │  ├─ numpy [1.26.0|1.26.2|1.26.3|1.26.4] would require
> │  │  └─ python_abi ==3.9 *_pypy39_pp73, which requires
> │  │     └─ python =3.9 *_73_pypy, which can be installed;
> │  ├─ numpy [1.26.0|1.26.2|1.26.3|1.26.4] would require
> │  │  └─ python >=3.10,<3.11.0a0 *, which can be installed;
> │  ├─ numpy [1.26.0|1.26.2|1.26.3|1.26.4] would require
> │  │  └─ python >=3.11,<3.12.0a0 *, which can be installed;
> │  ├─ numpy [1.26.0|1.26.2|1.26.3|1.26.4] would require
> │  │  └─ python_abi =3.12 *_cp312, which requires
> │  │     └─ python =3.12 *_cpython, which can be installed;
> │  └─ numpy [1.26.0|1.26.2|1.26.3|1.26.4] would require
> │     └─ python >=3.9,<3.10.0a0 *, which can be installed;
> └─ python =3.13 *_cp313 is not installable because there are no viable options
>    ├─ python [3.13.0|3.13.1] conflicts with any installable versions previously reported;
>    └─ python [3.13.0rc1|3.13.0rc2|3.13.0rc3] would require
>       └─ _python_rc =* *, which does not exist (perhaps a missing channel).
> ```

Any hint @goanpeca, @hmaarrfk ?

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by FrancescAlted** [comment]
Ok, so the thing is that we need to re-render whenever the recipe is touched, right?

**Comment by hmaarrfk** [comment]
> Ok, so the thing is that we need to re-render whenever the recipe is touched, right?

basically.....

it can't help.

it updates our tooling and ensures that package pinnings are compatible.

The "global" numpy pinning hasn't been touched in years, and can't be "guranteed" to be compatible with "bleeding edge" python (3.13)

**Comment by hmaarrfk** [comment]
lets try to build for other architecture.

**Comment by hmaarrfk** [comment]
thanks for the help! I built for arm + apple arm! too!

**Comment by FrancescAlted** [comment]
Ok. I tried this on my MacBook M1, but I am getting this error:

```
> conda install -c conda-forge python-blosc2                   (python-blosc2)
Channels:
 - conda-forge
 - defaults
Platform: osx-arm64
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - python-blosc2

Current channels:

  - https://conda.anaconda.org/conda-forge
  - defaults

```

Am I missing something?

**Comment by hmaarrfk** [comment]
Can you try again? Sometimes it takes time for the cloud to sync up

**Comment by hmaarrfk** [comment]
Am I not a maintainer on this already? 

I guess I’m not. 

But sure. Happy to help blocs

**Pull request merged**